tutordiscovery/patches/local-docker-compose-dev-services

```diff
@@ -4,8 +4,9 @@ discovery:
   command: ./manage.py runserver 0.0.0.0:8381
   stdin_open: true
   tty: true
-  {%- for mount in iter_mounts(MOUNTS, "discovery") %}
   volumes:
+    - ../../data/discovery-media:/openedx/discovery/course_discovery/media
```

Comment:
The pattern that we use for plugin data directories is `./../data/discovery/media`.